py/picca/pk1d/postproc_pk1d_oldoutput.py

```diff
@@ -0,0 +1,295 @@
+"""This module defines a set of functions to postprocess files produced by compute_pk1d.py.
```

Comment:
Yes, this file shall disappear at some point.
The point is, it would be a significant effort to merge both because the data structures are different, so the main function compute_mean_pk1d() is completely different in either case.

Marie Lynn kept the "old" file just for convenience, since she had scripts still using the old format. She's ok to remove the file if you think it's better, otherwise you may keep it for a few weeks before removing it in a later PR.

Reply:
File removed